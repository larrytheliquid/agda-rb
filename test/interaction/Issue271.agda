module Issue271 where

data D (A : Set) : Set where
  d : D A → D A

f : {A : Set} → D A → D A
f x = d {!!}
