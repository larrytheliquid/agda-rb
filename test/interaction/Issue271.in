top_command (cmd_load currentFile [])

-- The output should not contain superfluous outer parentheses.
goal_command 0 (cmd_goal_type_context Normalised) ""
