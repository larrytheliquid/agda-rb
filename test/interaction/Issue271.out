agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action "*All Goals*" "?0 : D .A
")
agda2_mode_code (agda2-goals-action '(0))
agda2_mode_code (agda2-status-action "")
agda2_mode_code (agda2-info-action
                 "*Goal type etc.*"
                 "Goal: D .A
————————————————————————————————————————————————————————————
x  : D .A
.A : Set")
agda2_mode_code (agda2-goals-action '(0))
